build.sh

```diff
@@ -42,7 +42,7 @@ docker push $GCR_IMAGE_REPO/$IMAGE_DIR:$TAG_NAME
 docker push $GCR_IMAGE_REPO/$IMAGE_DIR:$VERSION
 docker push $GCR_IMAGE_REPO/$IMAGE_DIR:latest
 
-if [[ $IMAGE_DIR = "terra-jupyter-aou" ]]; then
+if [[ $IMAGE_DIR = "terra-jupyter-aou" ]] ; then
```

Comment:
You'll probably want to add the added image dir here so we handle AOU images the same way